app/configurator/components/ui-helpers.ts

```diff
@@ -27,6 +27,12 @@ const parseDay = timeParse("%Y-%m-%d");
 const parseMonth = timeParse("%Y-%m");
 const parseYear = timeParse("%Y");
 
+const formatSecond = timeFormat("%Y-%m-%dT%H:%M:%S");
+const formatMinute = timeFormat("%Y-%m-%dT%H:%M");
+const formatDay = timeFormat("%Y-%m-%d");
+const formatMonth = timeFormat("%Y-%m");
+const formatYear = timeFormat("%Y");
```

Comment:
Thinking out loud if these shouldn't be connected to RDF definitions? See that we already defined some of them here: https://github.com/visualize-admin/visualization-tool/blob/92532b7e066e25b01ed719e841a79e70347ef8d9/app/rdf/parse.ts#L301-L305

Reply:
Yes, I was wondering if this would make sense and is actually needed. It looks that the formatters I shared in the link are only used in `interpolateTimeValues` function from the same file which is not used anywhere; so I guess we could just clean up and remove these server-side values 👀